example/android/app/src/main/kotlin/com/onegini/mobile/onegini_example/SecurityController.java

```diff
@@ -0,0 +1,7 @@
+package com.onegini.mobile.onegini_example;
+
+@SuppressWarnings({ "unused", "WeakerAccess" })
+public final class SecurityController {
+  public static final boolean rootDetection = false;
+  public static final boolean debugDetection = false;
```

Comment:
you can also add
```public static final boolean debugLogs = true;```
if we want to make devs life easier. We do that in our example app https://github.com/onewelcome/example-app-android/blob/master/app/src/main/java/com/onegini/mobile/exampleapp/SecurityController.java